src/transform.ts

```diff
@@ -397,6 +401,20 @@ function processSass(filename: string): string {
     sassLoadPathsConfig = [];
   }
 
+  // Workaround for ?bug? with sass.compileString that the location of the source file is not automatically included in load paths
+  sassLoadPathsConfig.push(path.parse(filename).dir);
+
+  const sharedSassResourcesPath = path.join(CACHE_FOLDER, SASS_SHARED_RESOURCES_CONCAT);
+  const buildOptions: BuildCssOptions = { prependUseShared: false};
+
+  if (fs.existsSync(sharedSassResourcesPath)) {
+    buildOptions.prependUseShared = true;
```

Comment:
@mikeb-meq I think we can move the logic to create `sassPrefixedWithSharedResources` to be here, the reasons should be:
- For consistency with `sassLoadPathsConfig`. We might update the `buildCssResult` arguments to `buildCssResult(sass: any, sassLoadPathsConfig: string[], sassPrefixedWithSharedResources: string, filename: string)
- Can remove BuildCssOptions